static/app/components/forms/controls/selectOption.tsx

```diff
@@ -56,17 +56,28 @@ function SelectOption(props: Props) {
           innerWrapProps={{'data-test-id': value}}
           labelProps={{as: typeof label === 'string' ? 'p' : 'div'}}
           leadingItems={
-            <Fragment>
-              <CheckWrap isMultiple={isMultiple} isSelected={isSelected}>
-                {isSelected && (
-                  <IconCheckmark
-                    size={isMultiple ? 'xs' : 'sm'}
-                    color={isMultiple ? 'white' : undefined}
-                  />
-                )}
-              </CheckWrap>
-              {data.leadingItems}
-            </Fragment>
+            itemProps.__isNew__ ? (
+              <Fragment>
+                <IconAdd size="sm" />
+                {data.leadingItems}
+              </Fragment>
```

Comment:
@Jesse-Box I added your suggestion here, but since I updated a core component, I would like to wait for @vuluongj20 opinion as well. 

it looks like this 
![image](https://github.com/user-attachments/assets/11463963-cbfc-4b91-acef-97405678a183)